webapp/src/Components/Player/Compare/PlayStyle/PlayerComparePlayStyleCharts.tsx

```diff
@@ -88,14 +80,39 @@ export class PlayerComparePlayStyleCharts extends React.PureComponent<Props, Sta
                 currentPlaylistsOnly
                 multiple={false}/>
         )
+
+        const rankSelect = (
+            <RankSelect selectedRank={this.state.rank || -1}
+                        handleChange={this.handleRankChange}
+                        inputLabel="Rank to compare"
+                        helperText="Select the rank to plot as average"
+                        noneLabel="Default"/>
+        )
+
+        if (playerPlayStyles.length === 0) {
+            return (
+	          <>
+                <Grid item xs={12} style={{textAlign: "center"}}>
```

Comment:
innards of fragment should be indented.

Reply:
I meant something like 
```
<>
    <Thing/>
</>
```

Reply:
Yeah, the tabs were screwing things up in github. Let's see if it worked this time.